<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1629886906913' stop='1629886921256'>
    <name>upload new dataset on myDatasets page..</name>
    <title>upload new dataset on myDatasets page..</title>
    <test-cases>
        <test-case start='1629886906913' status='failed' stop='1629886909743'>
            <name>clear old e2e projects</name>
            <title>clear old e2e projects</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='88017560-e00d-49f9-ae70-4c4fe34634f8-attachment.png' type='image/png' size='80849'/>
            </attachments>
            <failure>
                <message>Failed: each key must be a number of string; got object</message>
                <stack-trace>TypeError: each key must be a number of string; got object
                    at E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\webdriver.js:2162:21
                    at Array.forEach (&lt;anonymous>)
                    at E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\webdriver.js:2157:16
                    at processTicksAndRejections (node:internal/process/task_queues:96:5)
                From: Task: WebElement.sendKeys()
                    at Driver.schedule (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\webdriver.js:807:17)
                    at WebElement.schedule_ (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\webdriver.js:2010:25)
                    at WebElement.sendKeys (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\webdriver.js:2174:19)
                    at actionFn (E:\vmware\Loop\GitHub\e2e\node_modules\protractor\built\element.js:89:44)
                    at Array.map (&lt;anonymous>)
                    at E:\vmware\Loop\GitHub\e2e\node_modules\protractor\built\element.js:461:65
                    at ManagedPromise.invokeCallback_ (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:1376:14)
                    at TaskQueue.execute_ (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:2927:27Error: 
                    at ElementArrayFinder.applyAction_ (E:\vmware\Loop\GitHub\e2e\node_modules\protractor\built\element.js:459:27)
                    at ElementArrayFinder.&lt;computed> [as sendKeys] (E:\vmware\Loop\GitHub\e2e\node_modules\protractor\built\element.js:91:29)
                    at ElementFinder.&lt;computed> [as sendKeys] (E:\vmware\Loop\GitHub\e2e\node_modules\protractor\built\element.js:831:22)
                    at E:\vmware\Loop\GitHub\e2e\page-object\my-datasets-page.ts:32:48
                    at ManagedPromise.invokeCallback_ (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:1376:14)
                    at TaskQueue.execute_ (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:2927:27
                    at E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:668:7
                    at processTicksAndRejections (node:internal/process/task_queues:96:5)
                From: Task: Run it("clear old e2e projects") in control flow
                    at UserContext.&lt;anonymous> (E:\vmware\Loop\GitHub\e2e\node_modules\jasminewd2\index.js:94:19)
                    at attempt (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
                    at QueueRunner.run (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
                    at runNext (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
                    at E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4264:13
                    at E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4172:9
                    at E:\vmware\Loop\GitHub\e2e\node_modules\jasminewd2\index.js:64:48
                    at ControlFlow.emit (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\events.js:62:21)
                    at ControlFlow.shutdown_ (E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:2674:10)
                    at E:\vmware\Loop\GitHub\e2e\node_modules\selenium-webdriver\lib\promise.js:2599:53
                From asynchronous test: 
                Error: 
                    at Suite.&lt;anonymous> (E:\vmware\Loop\GitHub\e2e\features\clear-old-e2e-projects-datasets-spec.ts:20:3)
                    at addSpecsToSuite (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
                    at Env.describe (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
                    at describe (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
                    at Object.&lt;anonymous> (E:\vmware\Loop\GitHub\e2e\features\clear-old-e2e-projects-datasets-spec.ts:10:1)
                    at Module._compile (node:internal/modules/cjs/loader:1101:14)
                    at Module.m._compile (E:\vmware\Loop\GitHub\e2e\node_modules\ts-node\src\index.ts:422:23)
                    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
                    at Object.require.extensions.&lt;computed> [as .ts] (E:\vmware\Loop\GitHub\e2e\node_modules\ts-node\src\index.ts:425:12)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1629886909743' status='failed' stop='1629886921254'>
            <name>clear old e2e dataset</name>
            <title>clear old e2e dataset</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='bad1e5fa-09a2-45ac-a5e8-fb26867d0495-attachment.png' type='image/png' size='57403'/>
            </attachments>
            <failure>
                <message>Expected 3 to equal 0.</message>
                <stack-trace>Error: Failed expectation
                    at Object.&lt;anonymous> (E:\vmware\Loop\GitHub\e2e\features\clear-old-e2e-projects-datasets-spec.ts:47:59)
                    at Generator.next (&lt;anonymous>)
                    at fulfilled (E:\vmware\Loop\GitHub\e2e\node_modules\tslib\tslib.js:112:62)
                    at processTicksAndRejections (node:internal/process/task_queues:96:5)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>